src/Components/Tabs/Tabs.js

```diff
@@ -115,17 +116,20 @@ class Tabs extends Component {
   }
 
   getTabWidth(width) {
-    const { scrollEnabled, actionItems } = this.props;
+    const { scrollEnabled, actionItems, fixedTabWidth } = this.props;
+    const hasFixedTabWidth = !!fixedTabWidth;
+    let fixedWidth;
```

Comment:
`fixedWidth` is only ever used in the first if condition and is only set once, so no need for it be defined here or as a let